### ISSUE TITLE
Rotational shake?

### ISSUE THREAD

**Issue by johanhelsing** [johanhelsing/bevy_trauma_shake#2]
I'm wondering whether to add rotational shake or not. Personally, I'm not really sure how much I think it adds.

Curious to get some feedback.

**Comment by PraxTube** [comment]
Camera shake in my game [without rotation](https://youtu.be/iLSLr4MLPXs), [with rotation](https://youtu.be/mBdFj8ZcSQQ). I much prefer with rotation. You also saw the GDC talk right? I think he made a pretty good showcase there as well. I would at least give the user the option to add rotational shake if you don't want to always apply it.

**Comment by johanhelsing** [comment]
I think perhaps it depends on the kind of game. Happy to accept a pr that adds it in an optional way!

**Comment by PraxTube** [comment]
Yeah I agree. Haha, perhaps I will work on it once I rework my camera shake system.
